src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -61,17 +60,58 @@ std::string getEntities
   ParseData*                 parseDataP
 )
 {
-  std::string  answer;
-  Entities     entities;
+  using namespace std;
+  typedef vector<string>  vstr;
```

Comment:
Not sure I like these short names.
In my opinion, the code only gets more cryptic